internal/pkg/utils/testproject/project.go

```diff
@@ -289,15 +289,19 @@ func (p *Project) createBranches(branches []*fixtures.BranchState) {
 			}
 			p.setEnv(fmt.Sprintf("TEST_BRANCH_%s_ID", branch.Name), branch.Id.String())
 		} else {
-			p.storageApi.
+			err := p.storageApi.
 				CreateBranchRequest(branch).
 				OnSuccess(func(response *client.Response) {
 					p.logf(`crated branch "%s", id: "%d"`, branch.Name, branch.Id)
 					p.setEnv(fmt.Sprintf("TEST_BRANCH_%s_ID", branch.Name), branch.Id.String())
 					p.branchesById[branch.Id] = branch
 					p.branchesByName[branch.Name] = branch
 				}).
-				Send()
+				Send().
+				Err()
+			if err != nil {
+				assert.FailNow(p.t, fmt.Sprintf(`cannot create branch: %s`, err))
+			}
```

Comment:
@JakubMatejka, tento request sa neposiela cez `pool`, ale priamo, a nebola tu osetrena chyba.
Takze to neskor spadlo na `nil`.